xml/book_sles_containers.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet href="urn:x-suse:xslt:profiling:docbook50-profile.xsl"
+ type="text/xml"
+ title="Profiling step"?>
+<!DOCTYPE book
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+
+<!-- ##################################################################### -->
+<!-- SLE Container Guide                                                  -->
+<!-- ##################################################################### -->
+<?provo dirname="upgrade/"?>
+<book version="5.0" xml:lang="en" xml:id="book-sles-containers"
```

Comment:
We will have this ID as part of our URLs, and I'd also consider the option that we might have to publish this guide for openSUSE, so I'd suggest to use just "book-container" as the ID here. (cf. also https://documentation.suse.com/style/current/single-html/docu_styleguide/#sec-id -> use singular)

Reply:
Done. Thanks!